lib/Error/Card.php

```diff
@@ -27,4 +28,9 @@ public function getStripeParam()
     {
         return $this->stripeParam;
     }
+
+    public function getStripeDeclineCode()
+    {
+        return $this->stripeDeclineCode;
```

Comment:
@GrahamCampbell thanks for pointing that out, but I am following the style that is used in the other exception classes. Where the property requestId is not defined either. See this file for reference https://github.com/stripe/stripe-php/blob/master/lib/Error/Base.php#L21